pymc3/distributions/continuous.py

```diff
@@ -224,8 +224,9 @@ class Uniform(BoundedContinuous):
     """
 
     def __init__(self, lower=0, upper=1, *args, **kwargs):
-        self.lower = lower = tt.as_tensor_variable(floatX(lower))
-        self.upper = upper = tt.as_tensor_variable(floatX(upper))
+        # TODO: This does not show up on logpt :(
```

Comment:
This is the only thing I tried with the goal of having access to the name of the distribution parameters in the logpt graph